spring-cloud-vault-config-databases/src/test/java/org/springframework/cloud/vault/config/databases/CouchbaseSecretIntegrationTests.avaj

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2016-2020 the original author or authors.
```

Comment:
The file suffix is `avaj`.

Reply:
I have removed it. It should not have been there as this was for the secrets engine, not the database engine.